data.sql

```diff
@@ -8,3 +8,18 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
 VALUES ('Pikachu',DATE '2021-01-7',1,false,15.04);
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
 VALUES ('Devimon', DATE '2017-05-12',5,true,11);
+
+ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+ VALUES ('Charmander',DATE '2020-02-08',0,false,-11);
+ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+ VALUES ('Plantmon',DATE '2021-11-15',2,true,-5.7);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Squirtle',DATE '1993-04-02',3,false,-12.13);
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Angemon',DATE '2005-06-12',1,true,-45);
+ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+ VALUES ('Boarmon',DATE '2005-06-07',7,true,20.4);
+ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+ VALUES ('Blossom',DATE '1998-10-13',3,true,17);
+ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Ditto',DATE '2022-05-14',4,true,22);
```

Comment:
- [ ] You should create queries to answer the following questions in transactions: 👇🏻 

![19](https://user-images.githubusercontent.com/79058364/179806599-e2c56e8a-bece-4e06-bbcf-bd2e7992be26.png)

- Kindly address this 👍🏻

Reply:
It was actually in the dev branch, my mistake.